test_api.py

```diff
@@ -1,18 +1,23 @@
 from api_helper import ShoonyaApiPy
 import logging
+from dotenv import load_dotenv
  
 #enable dbug to see request and responses
 logging.basicConfig(level=logging.DEBUG)
 
 #start of our program
 api = ShoonyaApiPy()
 
-#credentials
-user    = <uid>
-pwd     = <password>
+#create .env file to securely access user credentials in program
+load_dotenv('/home/akash/Documents/Projects/.env')
```

Comment:
Please do not hardcode stuff for your local setup.

Reply:
even it better if it can initialize from user env. that means system variable

what do you think

Reply:
yes that way
app_key = os.getenv('shoonya_api_pwd')